media-libs/lib3mf/lib3mf-1.8.1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Implementation of the 3D Manufacturing Format file standard"
+HOMEPAGE="https://3mf.io/"
+SRC_URI="https://github.com/3MFConsortium/$PN/archive/v$PV.tar.gz -> $P.tar.gz"
+LICENSE="BSD"
+
+SLOT="0"
```

Comment:
Per ebuild.skel, `LICENSE` is in the same block as `SLOT`... that didn't need change from the previous state. ;)

Reply:
Oh yes, you're right, there could be S inbetween SRC_URI and LICENSE. 
Although in my opinion, LICENSE is logically more tied to the block with SRC_URI. The block containing KEYWORDS and SLOT are metadata which is related to building... where (arch) and what (IUSE, SLOT, RESTRICT) to build. And many ebuilds have in in the upper block ;)
I'm gonna change it back.